sdk/selfhelp/Azure.ResourceManager.SelfHelp/tests/TroubleshooterTests.cs

```diff
@@ -13,8 +13,10 @@ namespace Azure.ResourceManager.SelfHelp.Tests
 
     public class TroubleshooterTests : SelfHelpManagementTestBase
     {
-        public TroubleshooterTests(bool isAsync) : base(isAsync)//, RecordedTestMode.Record)
+        private bool isAsync = true;
+        public TroubleshooterTests(bool isAsync) : base(isAsync, RecordedTestMode.Record)
```

Comment:
You should not hard-code `RecordedTestMode` here. It should be determined by the environment variable `AZURE_TEST_MODE`. By default it's `Playback`. If you have problems in playback, you need to check the test cases to see if they are repeatable.